app/processing/requires-manual-ledger-check.js

```diff
@@ -0,0 +1,17 @@
+const util = require('util')
+const getCompletedPaymentRequests = require('./get-completed-payment-requests')
+
+const requiresManualLedgerCheck = async (paymentRequest) => {
+  console.log(`Payment requests: ${util.inspect(paymentRequest, false, null, true)}`)
+  let requiresManualCheck = false
+  if (paymentRequest.value <= 0) {
+    requiresManualCheck = true
```

Comment:
Remove the `let` and change to `return true`

Early return pattern is advised for Javascript due to its async nature, reducing the requirement for `else` statements plus it helps to differentiate between edge case

Reply:
SonarQube flagged two early return true as code smell, it advised they should be combined as single return.